contrib/kind-helm.sh

```diff
@@ -294,25 +325,64 @@ EOT
     fi
 }
 
+label_ovn_single_node_zones() {
+  KIND_NODES=$(kind get nodes --name "${KIND_CLUSTER_NAME}")
```

Comment:
please use kind_get_nodes . it factors in cases when kind creates additional nodes like load_balancer.

Ref: https://github.com/ovn-org/ovn-kubernetes/blob/ff286af7048c286f77963490f4756516da6ee9b0/contrib/kind-common#L438-L440

Reply:
fixed.